lib/components/transaction_item.dart

```diff
@@ -149,13 +106,17 @@ class TransactionItem extends StatelessWidget {
                 child: Text(l10n.transactionItemButtonCopyToClipboard),
               ),
               // sally: commenting this temporary until we improve the resend to support all transfers
-              //if (appStore.currentQubicIDs
-              //      .any((e) => e.publicId == item.sourceId) &&
-              //item.getStatus() != ComputedTransactionStatus.pending)
-              //PopupMenuItem<CardItem>(
-              //value: CardItem.resend,
-              //child: Text(l10n.transactionItemButtonResend),
-              //),
+              if (appStore.currentQubicIDs
```

Comment:
@ahmed-tarek-salem 
1. my comment does not apply anymore, please delete it
2. Resend only should be available for the non watch only accounts

Reply:
@sallymoc 

The watch-only account doesn't have an option for sending transactions, so it's excluded by default.

Reply:
That is not what I see in my test

Reply:
![image](https://github.com/user-attachments/assets/a1572866-1c0d-421c-a34d-a8e1b8ccbff6)

Reply:
Yeah got it, resolved.